client/main.lua

```diff
@@ -335,10 +335,10 @@ local function ParkOwnedVehicle(veh, garageName, vehLocation, plate)
 
     local totalFuel = 0
 
-    if config.FuelScript then
-        totalFuel = exports[config.FuelScript]:GetFuel(veh)
-    elseif config.FuelScript == '' then
+    if config.FuelScript == '' then
         totalFuel = Entity(veh).state.fuel
+    elseif config.FuelScript then
+        totalFuel = exports[config.FuelScript]:GetFuel(veh)
```

Comment:
```suggestion
    if config.FuelScript ~= '' then
        totalFuel = exports[config.FuelScript]:GetFuel(veh)
    else
        totalFuel = Entity(veh).state.fuel
```

Little bit simpler and cleaner way to write this.